draft-ZIP-0227.rst

```diff
@@ -0,0 +1,184 @@
+
+The main requirement for the asset identifier is for it to be collision-free, as two different issuers should not be able to issue the same asset type. As such we need the asset identifier to be:
+
+- pseudorandom
+- deterministic
+- dependent on the issuer ID
+
+We define the asset identifier `assetID` (represented as a string) as
+
+:math:`\mathsf{assetID := GroupHash^{\mathbb{P}}}\mathsf{(issuerID || assetDesc)}`
+
+where
+
+- `assetDesc` is the asset description, which includes any information pertaining to the issuance.
+- `issuerID` is the public key of the issuer, used to verify the signature on the transaction and is defined as `ik` (issuer key or issuer verification key):
+
+
+:math:`\mathsf{ik := SpendAuthSig^{Orchard}.DerivePublic(isk)}`
+
+
+which uses the same signature as the spend authorization signature, where `isk` is the issuer secret key. The verification key is derived from `isk`. In order to differentiate the `isk` and `ik` keys from the actual spend authorization key, we derive another branch of keys from the main spending key `sk` as such:
+
+:math:`\mathsf{isk := ToScalar^{Orchard}(︀ PRF^{expand}_{sk} ([10])}`
+
+This allows the issuer to use the same wallet it usually uses to transfer assets, while keeping a disconnect from the other keys. It provides further anonymity and the ability to delegate issuance of an asset (or in the future, generate a multi-signature protocol) while the rest of the keys in the wallet safe.
+
+Then we let `issuanceInfo := (assetID, finiteSupply)`, where 
+
+- `assetID` is as defined above.
+- `finiteSupply` is a boolean variable that defines whether this issuance transaction of the specific asset type is the final issuance of tokens or not. Once `finiteSupply` is set, it cannot be unset. This allows expanding the functionality of the issuance mechanism:
+    - Provides `assetID` revocation in case of compromise (issue last issuance with 0 token value)
+    - Supports NFT issuance (where the first issuance with value of 1 is also the last)
+
+
+Issuance Protocol
+=================
+
+The issuance protocol allows for a single issuance to be sent to many receivers, as the issuerID does not have to match the address ownership of the notes output. Furthermore, every transaction can contain many issuance instances. The design presented in this ZIP enables several use cases for issuance of shielded assets.
+
+- The issuer knows in advance the receivers of the issued asset.
+- The asset is of non-fungible type, where each asset type can be made part of a single “series”
+- The supply of the asset is limited or not
+- The assets can be wrapped versions of assets in other chains (as long as there is a bridge that supports it)
```

Comment:
Move to Overview/Motivation?

Reply:
This changed with the new PR, please see there.